app/helpers.php

```diff
@@ -245,3 +245,15 @@ function contact_information(): string
         );
     }
 }
+
+if (! function_exists('orientation_link')) {
+    function orientation_link(string $userType): string
+    {
+        return match ($userType) {
+            'individual' => 'https://share.hsforms.com/161eyaBsQS-iv1z0TZLwdQwdfpez',
+            'organization' => 'https://share.hsforms.com/1sB6UV4gvQlC_0QxQ3q3z1Adfpez',
+            'regulated-organization' => 'https://share.hsforms.com/1gGf9TjhaQ0uaqcnyJfSDlwdfpez',
```

Comment:
Can we get the keys from the UserContext enum so that we guarantee that it matches?